lib/protocol/deviceid.go

```diff
@@ -15,6 +15,7 @@ import (
 
 const (
 	DeviceIDLength      = 32
+	// keep consistent with shortIDStringLength in gui/default/syncthing/app.js
 	ShortIDStringLength = 7
```

Comment:
This is the source of the CI failure: The added comment changes the alignment `gofmt -S` expects:

```
=== RUN   TestCheckGoFmt
    gofmt_test.go:39: File ../lib/protocol/deviceid.go is not formatted correctly:
        
        diff ../lib/protocol/deviceid.go.orig ../lib/protocol/deviceid.go
        --- ../lib/protocol/deviceid.go.orig
        +++ ../lib/protocol/deviceid.go
        @@ -14,7 +14,7 @@
         )
         
         const (
        -	DeviceIDLength      = 32
        +	DeviceIDLength = 32
         	// keep consistent with shortIDStringLength in gui/default/syncthing/app.js
         	ShortIDStringLength = 7
         )
--- FAIL: TestCheckGoFmt (1.20s)
```

https://github.com/syncthing/syncthing/actions/runs/7387700143/job/20096964099?pr=9314

Reply:
@imsodin Fixed :+1: